Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,47 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            int entry = 1;
+            const int maxEntries = 3;
+
+            while (entry < maxEntries)
```

Comment:
This would be the perfect opportunity for a `for` loop :)

`for(int i = 0; i<maxEntries; i++)`